components/mpas-seaice/docs/user-guide/index.md

```diff
@@ -0,0 +1,87 @@
+User's Guide
+============
+
+Guidance for using E3SM is available from [E3SM's public web site](<https://e3sm.org/model/running-e3sm/e3sm-quick-start>).
+
+**MPAS Framework**
```

Comment:
This section seems to belong in the Developers guide.

Reply:
I could add it there too, but I figure the developer will need to be a user first...

Reply:
Yes I meant the MPAS Framework section.   Its not something a user **needs** to know to use the model.

Reply:
You don't have to do it in this PR but this could instead go in components/mpas-framework/docs. Especially if its relevant for all the MPAS models.

Reply:
Sure that's fine.